app/controllers/static_pages_controller.rb

```diff
@@ -1,7 +1,16 @@
 class StaticPagesController < ApplicationController
 
   def index
-    # added to pass initial rspec case
+  end
+
+
+  def mygames
+    @mygames = get_mygames
```

Comment:
There is a nice shortcut for doing this in a slightly more efficient way:

```
@mygames ||= get_mygames
```

this will make sure that you don't run `get_mygames` every time you call `mygames`, but only the first time.. it will set `@mygames` and if you run it the second time, it will just return `@mygames`

Reply:
thanks, will make that change then merge!